charts/llm-d-modelservice/templates/epp-deployment.yaml

```diff
@@ -19,60 +19,24 @@ spec:
       containers:
       - name: epp
         imagePullPolicy: Always
-        image: {{ default "ghcr.io/llm-d/llm-d-inference-scheduler:0.0.3" .Values.endpointPicker.image }}
+        image: {{ default "ghcr.io/llm-d/llm-d-inference-scheduler:0.0.3" .Values.routing.epp.image }}
         args:
         - --poolName
         - {{ include "llm-d-modelservice.fullname" . }}-inference-pool
```

Comment:
We need an optional override here for passing the name of an existing inferencepool if we are not creating the inferencepool - this is the composability concern

Reply:
good catch!